includes/wc-core-functions.php

```diff
@@ -442,6 +442,7 @@ function get_woocommerce_currencies() {
 					'UYU' => __( 'Uruguayan peso', 'woocommerce' ),
 					'UZS' => __( 'Uzbekistani som', 'woocommerce' ),
 					'VEF' => __( 'Venezuelan bol&iacute;var', 'woocommerce' ),
+					'VES' => __( 'Bolívar soberano', 'woocommerce' ),
```

Comment:
We need to convert HTML entities.
Please, replace `Bolívar` to `Bol&iacute;var`